### PR TITLE
Unversion versioned table before dropping it

### LINKED ISSUE #180: DROP TABLE CASCADE is not supported anymore by table_version starting 1.6.0
The DROP TABLE command we're issuing is not supported anymore by newer `table_version`, starting version 1.6.0 (to fix https://github.com/linz/postgresql-tableversion/pull/149).

We now need to first UNVERSION the table, and then DROP it (until we'll stop dropping as a whole, which is what #148 asks for)

### PR DESCRIPTION
**Pull request by strk**
Closes #180

### CODE REVIEW

**Comment by strk** [comment]
Before merging this PR I want to make sure Travis would be catching the issue

File: sql/02-bde_control_functions.sql.in [hunk 1]
```diff
@@ -1966,7 +1966,12 @@ BEGIN
         RAISE INFO '%', v_task;
 
         PERFORM bde_control._bde_GetExclusiveLock(p_upload,v_bdetable);
-        v_sql := 'DROP TABLE ' || v_bdetable || ' CASCADE';
+
+        v_sql := 'PERFORM table_version.ver_disable_versioning( ' || v_bdetable || ')';
```
Comment (palmerj) [inline sql/02-bde_control_functions.sql.in]: Can we please do a check to ensure the table version support is present before calling this API.

**Comment by strk** [comment]
Ok in its current status Travis doesn't seem to be testing the path of bug #180, that is, it's probably NEVER loading a LEVEL0 update against a versioned table. I want to add this test before merging the fix

**Comment by strk** [comment]
Ok I'm now happy. First commit adds the test,which fails, second commit fixes the code so the test passes. @palmerj I answered your comment, ready to approve ?

**Comment by palmerj** [comment]
I'm still seeing CI failures

**Comment by strk** [comment]
Travis must be broken, if you see the icon next to the latest commit it is green (at least here) - I'll try restarting the build

**Pull request merged**